app/src/main/assets/index/lb.json

```diff
@@ -168,5 +168,357 @@
       "lat": "45.497111",
       "lon": "-73.578065"
     }
+  ],
+  "nodes": [
```

Comment:
What follows is a huge list containing all the "nodes" of the graph representing all hallways in LB floors 2 and 3.
X and Y are between 0 and 1 because they're relative to the topleft corner of the library, this was easier than manually entering all the coordinates (took forever to look them up on google maps).

Don't worry about nodes having unidirectional edges, when building the graph it ensures that all edges are bidirectional (there is a unit test to verify that postcondition)